homeassistant/components/group/config_flow.py

```diff
@@ -183,6 +184,11 @@ async def _set_group_type(
         preview="group",
         validate_user_input=set_group_type("binary_sensor"),
     ),
+    "climate": SchemaFlowFormStep(
+        basic_group_config_schema("climate"),
+        preview="climate",
```

Comment:
I didn't see any preview method in the climate entity

Reply:
Added the preview and also test it